### ISSUE TITLE
[0.5] Disabled Input/Select is difficult to read

### ISSUE THREAD

**Issue by startailcoon** [bitshares/bitshares-ui#2516]
A Disabled Input or Select will be difficult to read by the user.

- [ ] Find a background color + text color that works on all themse for disabled fields that can still be read easily.

## Example
![bild](https://user-images.githubusercontent.com/12114550/54074653-76129380-4295-11e9-9bb4-f5e1f58b1f20.png)

**Comment by froooze** [comment]
>     * [ ]  Find a background color + text color that works on all themse for disabled fields that can still be 

Same color for light and dark template? Does this make sense?

**Comment by sschiessl-bcp** [comment]
> > ```
> > * [ ]  Find a background color + text color that works on all themse for disabled fields that can still be 
> > ```
> 
> Same color for light and dark template? Does this make sense?

I agree that is an unnecessary restriction. Contrast can be increased, but musnt be same for all templates.

**Comment by startailcoon** [comment]
Sorry for being vague, I meant that we should find a color (depending on theme) that works for disabled fields. So no restriction to find just one for all.
